core/src/impl/Kokkos_Profiling.hpp

```diff
@@ -295,11 +327,22 @@ struct ComplexReducerSizeCalculator {
   }
 };
 
+template <typename Policy>
+auto default_tuned_version_of(const Policy& policy) {
+  return policy;
+}
+template <class... Properties>
+auto default_tuned_version_of(
+    const Kokkos::RangePolicy<Properties...>& policy) {
+  return Kokkos::Experimental::prefer(
+      policy, Kokkos::Experimental::DesiredOccupancy(100));
+}
```

Comment:
Suppose tuning is enabled, but no tuning tool is loaded. We still need to return a tuned policy, though. I think saying "Maximize Occupancy" is valid, here?

Reply:
`DesiredOccupcany` has a constructor taking `MaximizeOccupancy` if that's what you are asking.

Reply:
Oh cool, I'll use that

Reply:
Turns out, the type returned by that MaximizeOccupancy is different than the one from using DesiredOccupancy 100. For consistency, I'll use DesiredOccupancy 100